test/verifiedTests/testpFBA/testpFBA.m

```diff
@@ -0,0 +1,152 @@
+% The COBRAToolbox: testpFBA.m
+%
+% Purpose:
+%     - tests the basic functionality of pFBA
+%       Tests the basic solution for both minimizing the flux of gene-
+%       associated reactions and all rxns, while growing on gluose or lactose
+%       minimal media. Does not test the functionality of the map function.
+%
+% Authors:
+%     - Original file: Nathan Lewis 08/30/10
+%     - CI integration: Laurent Heirendt February 2017
+%
+% Note:
+%     - The solver libraries must be included separately
+
+% define global paths
+global path_TOMLAB
+global path_GUROBI
+
+% define the path to The COBRAToolbox
+pth = which('initCobraToolbox.m');
+CBTDIR = pth(1:end - (length('initCobraToolbox.m') + 1));
+
+initTest([CBTDIR, filesep, 'test', filesep, 'verifiedTests', filesep, 'testpFBA']);
+
+%tolerance
+tol = 1e-8;
+
+% load models and expected results
+load('testpFBAData.mat', 'model_glc', 'model_lac');
+og_ = load('testpFBAData.mat', 'GeneClasses_glc1', 'GeneClasses_glc0', 'GeneClasses_lac1', 'GeneClasses_lac0');
```

Comment:
The names for the following variables `og_`, `or_` and `om_` are a little bit unusual. Please change them.